docs/required-trino-apis.md

```diff
@@ -33,9 +33,9 @@ to call `StdUF#init()` and pass the `FunctionDependencies` and `FunctionBinding`
 ```java
 @Override
 public ScalarFunctionImplementation specialize(FunctionBinding functionBinding, FunctionDependencies functionDependencies) {
-  StdFactory stdFactory = new TrinoFactory(functionBinding, functionDependencies);
-  StdUDF stdUDF = getStdUDF();
-  stdUDF.init(stdFactory);
+  StdFactory typeFactory = new TrinoFactory(functionBinding, functionDependencies);
```

Comment:
This still uses `StdFactory`.

Reply:
Do you want to update the doc right now? I planned to update that after all class and method change. anyway.

Reply:
an we skip commenting on .md file change right now? After we change all API class name and method name, we still need to review all docs + the hyperlink in those docs. Do you agree? Right now, all md file changes are trigged by the IntelliJ automatically. The full change requires no only the class name change in sample code but also the official hyperlink of the classes in github location. I worry that can be out of updated after we 'renaming the branch'. It had better to be done after renaming the branch.